webtau/src/main/java/com/twosigma/webtau/http/config/WebTauHttpConfiguration.java

```diff
@@ -21,19 +21,21 @@
 import com.twosigma.webtau.http.HttpUrl;
 
 public class WebTauHttpConfiguration implements HttpConfiguration {
-    private WebTauConfig cfg = WebTauConfig.INSTANCE;
-
     @Override
     public String fullUrl(String url) {
         if (HttpUrl.isFull(url)) {
             return url;
         }
 
-        return HttpUrl.concat(cfg.getBaseUrl(), url);
+        return HttpUrl.concat(getCfg().getBaseUrl(), url);
     }
 
     @Override
     public HttpRequestHeader fullHeader(HttpRequestHeader given) {
         return given;
     }
+
+    private WebTauConfig getCfg() {
```

Comment:
Should we rename `WebTauConfig.getInstance()` to `WebTauConfig.getCfg()` then you can use `getCfg()` everywhere with a static import without having to define an identical private `getCfg()` function everywhere.

Alternatively define a static public function `getCfg()` in some helper class so we can avoid the unnecessary duplication.